supplementary_style_guide/glossary_terms_conventions/general_conventions/c.adoc

```diff
@@ -141,7 +141,7 @@
 [discrete]
 [[ceph-block-device]]
 ==== image:images/yes.png[yes] Ceph Block Device (noun)
-*Description*: In Red Hat Ceph Storage, the Ceph Block Device is the block storage component of Ceph. Also known as the RADOS Block Device, however the term Ceph Block Device is preferred.
+*Description*: In Red Hat Ceph Storage, the _Ceph Block Device_ is the block storage component of Ceph. Also known as the _RADOS Block Device_, however the term _Ceph Block Device_ is preferred.
```

Comment:
Another thought that the last instance of "Ceph Block Device" should be in quotes

Reply:
:heavy_check_mark: